include/seqan3/core/pod_tuple.hpp

```diff
@@ -6,371 +6,12 @@
-}
-//!\}
-
-} // namespace seqan3
-
-namespace std
-{
-
-//!\cond
-template <std::size_t i, typename ...types>
-constexpr auto & get(seqan3::pod_tuple<types...> & t) noexcept
-    requires (i < sizeof...(types))
-{
-    return seqan3::get<i>(t);
-}
-
-template <std::size_t i, typename ...types>
-constexpr auto const & get(seqan3::pod_tuple<types...> const & t) noexcept
-    requires (i < sizeof...(types))
-{
-    return seqan3::get<i>(t);
-}
-
-template <std::size_t i, typename ...types>
-constexpr auto && get(seqan3::pod_tuple<types...> && t) noexcept
-    requires (i < sizeof...(types))
-{
-    return seqan3::get<i>(std::move(t));
-}
-
-template <std::size_t i, typename ...types>
-constexpr auto const && get(seqan3::pod_tuple<types...> const && t) noexcept
-    requires (i < sizeof...(types))
-{
-    return seqan3::get<i>(std::move(t));
-}
-
-template <typename type, typename ...types>
-constexpr auto & get(seqan3::pod_tuple<types...> & t) noexcept
-    requires (seqan3::pack_traits::count<type, types...> == 1)
-{
-    return seqan3::get<type>(t);
-}
-
-template <typename type, typename ...types>
-constexpr auto const & get(seqan3::pod_tuple<types...> const & t) noexcept
-    requires (seqan3::pack_traits::count<type, types...> == 1)
-{
-    return seqan3::get<type>(t);
-}
-
-template <typename type, typename ...types>
-constexpr auto && get(seqan3::pod_tuple<types...> && t) noexcept
-    requires (seqan3::pack_traits::count<type, types...> == 1)
-{
-    return seqan3::get<type>(std::move(t));
-}
-
-template <typename type, typename ...types>
-constexpr auto const && get(seqan3::pod_tuple<types...> const && t) noexcept
-    requires (seqan3::pack_traits::count<type, types...> == 1)
-{
-    return seqan3::get<type>(std::move(t));
-}
-//!\endcond
-
-/*!\brief Obtains the type of the specified element.
- * \implements seqan3::transformation_trait
- * \relates seqan3::pod_tuple
- * \see [std::tuple_element](https://en.cppreference.com/w/cpp/utility/tuple/tuple_element)
- */
-template <std::size_t i, template <typename ...> typename t, typename ...types>
-//!\cond
-    requires (i < sizeof...(types)) &&
-            std::is_base_of_v<seqan3::pod_tuple<types...>, t<types...>>
-//!\endcond
-struct tuple_element<i, t<types...>>
-{
-    //!\brief Element type.
-    using type = seqan3::pack_traits::at<i, types...>;
-};
-
-/*!\brief Provides access to the number of elements in a tuple as a compile-time constant expression.
- * \implements seqan3::unary_type_trait
- * \see std::tuple_size_v
- * \relates seqan3::pod_tuple
- */
-template <template <typename ...> typename t, typename ...types>
-    requires std::is_base_of_v<seqan3::pod_tuple<types...>, t<types...>>
-struct tuple_size<t<types...>> :
-    public std::integral_constant<std::size_t, sizeof...(types)>
-{};
-
-} // namespace std
+SEQAN3_DEPRECATED_HEADER(
+"This header is deprecated and will be removed in SeqAn-3.1.0 Please #include <seqan3/utility/tuple/pod_tuple.hpp> instead.")
```

Comment:
You are missing a new line here (files always end with a new line). All editor usually have a setting for this somewhere :)

Reply:
Fixed it! Thanks for noticing :)